gradlew

```diff
@@ -197,11 +198,15 @@ if "$cygwin" || "$msys" ; then
     done
 fi
 
-# Collect all arguments for the java command;
-#   * $DEFAULT_JVM_OPTS, $JAVA_OPTS, and $GRADLE_OPTS can contain fragments of
-#     shell script including quotes and variable substitutions, so put them in
-#     double quotes to make sure that they get re-expanded; and
-#   * put everything else in single quotes, so that it's not re-expanded.
+
+# Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
we can add JVM options in gradle.properties file

Please see this once: https://github.com/hmcts/rd-professional-api/blob/master/gradle.properties

Reply:
This is auto-generated (downloaded) file while running `./gradlew wrapper --gradle-version latest` command to update gradle wrapper.